src/components/PageHeader.astro

```diff
@@ -13,7 +13,7 @@ const { pageKey } = Astro.props as PageHeaderProps;
 ---
 
 <div
-  class="flex flex-col flex-auto content-center justify-start items-center pointer-none"
+  class="flex flex-col flex-auto content-center justify-start items-center pointer-none w-fit"
```

Comment:
This makes sure the `PageHeader`s don't grow full width when I make the content larger than them